core/assets/bundles/bundle.properties

```diff
@@ -14,13 +14,20 @@ screenshot = Screenshot saved to {0}
 screenshot.invalid = Map too large, potentially not enough memory for screenshot.
 gameover = Game Over
 gameover.pvp = The[accent] {0}[] team is victorious!
+gameover.nextmap = Next selected map: {0}
+gameover.nextgame = New game begins in {0} seconds.
+gameover.mapby = by [accent]{0}
+gameover.eliminated = Your team got eliminated!
+gameover.place = You placed [accent]#{0}
```

Comment:
# characters are comments in properties files; you should escape this with a backslash.